src/videodownlinkbandwidthpolicy/VideoPriorityBasedPolicy.ts

```diff
@@ -353,6 +353,14 @@ export default class VideoPriorityBasedPolicy implements VideoDownlinkBandwidthP
         NetworkEvent.Decrease,
         numberOfParticipants
       );
+
+      // to prevent from instant drop due to the first-time dip
+      if (
+        this.startupPeriod &&
```

Comment:
Doesn't this still run into the issue where when you are out of the startup period, it will instantly drop down if there hasn't been a subscribe in a while?